snippets/javascript/nuxt/nuxt.json

```diff
@@ -327,506 +31,57 @@
     "description": "nuxt-named-view element",
     "scope": "vue-html"
   },
-  "nuxt-data": {
-    "prefix": ["nuxt data", "vdata"],
-    "body": ["data() {", "\treturn {", "\t\t${1:key}: ${2:value}", "\t}", "},"],
-    "description": "Vue Component Data",
-    "scope": "vue"
-  },
-  "nuxt-methods": {
-    "prefix": ["nuxt methods", "vmethod"],
-    "body": ["methods: {", "\t${1:name}() {", "\t\t${0}", "\t}", "},"],
-    "description": "vue method",
-    "scope": "vue"
-  },
-  "nuxt-computed": {
-    "prefix": ["nuxt computed", "vcomputed"],
-    "body": [
-      "computed: {",
-      "\t${1:name}() {",
-      "\t\treturn this.${2:data} ${0}",
-      "\t}",
-      "},"
-    ],
-    "description": "computed value",
-    "scope": "vue"
+  "nuxt-nuxtchild": {
+    "prefix": ["nuxt nuxtchild", "nuxtchild"],
```

Comment:
I think we can have just one prefix and name it nuxt child, don’t think there is a need of repeating nuxt word twice. Please take a look at snippets above too.